docs/source/src/python/user-guide/transformations/joins.py

```diff
@@ -1,3 +1,28 @@
+import requests
+
+
+DATA = [
+    (
+        "https://raw.githubusercontent.com/pola-rs/polars-static/refs/heads/master/data/monopoly_props_groups.csv",
+        "docs/assets/data/monopoly_props_groups.csv",
+    ),
+    (
+        "https://raw.githubusercontent.com/pola-rs/polars-static/refs/heads/master/data/monopoly_props_prices.csv",
+        "docs/assets/data/monopoly_props_prices.csv",
+    ),
+]
+
+
+for url, dest in DATA:
```

Comment:
Maybe check if data already exist and only download in case the files are missing?

Reply:
Of course 🤦  done in 95c6b99